docs/src/developing/lookup-tables.md

```diff
@@ -130,18 +130,13 @@ transactionV0.sign([payer]);
 
 // send and confirm the transaction
 // (NOTE: There is NOT an array of Signers here; see the note below...)
-const txid = await web3.sendAndConfirmTransaction(connection, transactionV0);
+const txid = await connection.sendTransaction(transactionV0);
```

Comment:
I think this should be updated to `web3.sendAndConfirmTransaction(connection, transactionV0, [payer])` to work correctly.

Reply:
okay. let me create an issue there

Reply:
hey @mcintyre94 do you have any thoughts on this?